.scala-steward.conf

```diff
@@ -15,5 +13,9 @@ dependencyOverrides = [
   {
     dependency = { groupId = "software.amazon.awssdk" },
     pullRequests = { frequency = "7 day" }
+  },
+  {
+    dependency = { groupId = "com.google.apis" },
+    pullRequests = { frequency = "7 day" }
```

Comment:
I don't think we're using `com.google.apis` in this project? The library dependencies are all listed here: https://github.com/guardian/grid-feeds/blob/main/build.sbt#L37